test/com/lynxanalytics/biggraph/frontend_operations/NetworKitTest.scala

```diff
@@ -32,7 +32,7 @@ class NetworKitTest extends OperationsTestBase {
         "Betweenness (estimate)" -> Map(0 -> 0.0, 1 -> 0.0, 2 -> 0.0, 3 -> 0.0),
         "Eigenvector" -> Map(0 -> 0.7, 1 -> 0.7, 2 -> 0.0, 3 -> 0.0),
         "Harmonic Closeness" -> Map(0 -> 0.33, 1 -> 0.33, 2 -> 0.66, 3 -> 0.0),
-        "Katz" -> Map(0 -> 0.5, 1 -> 0.5, 2 -> 0.49, 3 -> 0.49),
+        "Katz" -> Map(0 -> 0.7, 1 -> 0.7, 2 -> 0.06, 3 -> 0.06),
```

Comment:
So I guess this is the line that the commit title refers to? The rest is just minor refactor not worthy of a mention... ;)

Reply:
It's fine of course and I do like this "minor refactor" a lot.

Reply:
> So I guess this is the line that the commit title refers to? The rest is just minor refactor not worthy of a mention... ;)

Huh, good catch! I thought I committed this separately? Fixed in a rebase now.